src/components/app/data/services/bffs.js

```diff
@@ -0,0 +1,34 @@
+import { getConfig } from '@edx/frontend-platform/config';
+import { logError } from '@edx/frontend-platform/logging';
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+import { camelCaseObject } from '@edx/frontend-platform/utils';
+
+export const learnerDashboardBFFResponse = {
+  enterpriseCustomerUserSubsidies: {
+    subscriptions: {
+      customerAgreement: {},
+      subscriptionLicenses: [],
+      subscriptionLicensesByStatus: {},
+    },
+  },
+  enterpriseCourseEnrollments: [],
+  errors: [],
+  warnings: [],
+};
+
+export async function fetchEnterpriseLearnerDashboard(enterpriseId, enterpriseSlug, lmsUserId) {
```

Comment:
The BFF API relies on the request user (at least for now); wouldn't do anything with the `lms_user_id` param.